compiler/src/iree/compiler/InputConversion/MHLO/MHLOToMHLOPreprocessing.cpp

```diff
@@ -1296,9 +1301,10 @@ struct MHLOToMHLOPreprocessingPass
     patterns.insert<ExpandRngNormal, MulCastOfBool>(context);
 
     // scatter canonicalization patterns
-    patterns.insert<ScatterOpImplicitIndex, ScatterOpImplicitBatch,
-                    ScatterMaterializeInsertedDim, ScatterOpCollapseBatch>(
-        context);
+    patterns.insert<ScatterOpImplicitIndex>(context);
```

Comment:
This change seems unrelated (the two forms here are equivalent)

Reply:
Oops, this was some debugging code to track down the offending pass. Reverted.